scania-digital-design-system/sdds
# fix(badge): badge fixes

**Describe pull-request**  
Renamed story and removed dependency to preview-html styles

**Solving issue**  
Fixes: [AB#2650](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2650) [AB#2651](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2651) [AB#2655](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2655)

**How to test**  
_Add description how to test if possible_
1. Go to storybook link below
2. Check in Components -> Badge
3. Check the controls and preview template

**Screenshots**  
-

**Additional context**  
-
